build.sbt

```
@@ -7,6 +7,7 @@ lazy val scmUrl                 = "https://github.com/sky-uk/kafka-topic-loader"
 
 name                   := "kafka-topic-loader"
 organization           := "uk.sky"
+description            := "Loads the state of Kafka topics to populate your application on startup"
```

```suggestion
description            := "Reads the contents of provided Kafka topics"
```

Think yours is a little too specific

---

👍  I grabbed it from the Github description, do you think we should update that one too?

---

Yeah i think that would make sense